src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -146,7 +147,9 @@ class CodeLongevity(private val serverRepo: Repo,
                     git: Git) {
     val repo: Repository = git.repository
     val head: RevCommit =
-        RevWalk(repo).parseCommit(repo.resolve(RepoHelper.MASTER_BRANCH))
+        try { RevWalk(repo).parseCommit(repo.resolve("HEAD")) }
+        catch(e: Exception) { throw Exception("No default branch") }
```

Comment:
No master branch. But existence of master branch checked before run of hasher, so this check is excess.

Reply:
np, I'll revert the branch name change. However I think this requirement is too restrictive though and overcomplicates the things in general. I'll file a bug on it to have further discussion.

Reply:
btw, I'd like to keep the try/catch thing if that was your point, because it's useful for testing outside the main infrastructure.